system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -43,6 +44,58 @@
                            (map #(select-keys % [:concept-id :extra-fields :metadata])))]
     (#'jobsub/process-subscriptions (system/context) subscriptions)))
 
+(deftest subscription-count-exceeds-limit-test
+  (side/eval-form `(jobsub/set-subscriptions-limit! 1))
+  (let [coll1 (data-core/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection
+                {:ShortName "coll-exceed-limit-1"
+                 :EntryTitle "entry-title-exceed-limit-1"})
+               {:token "mock-echo-system-token"})
+        coll2 (data-core/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection
+                {:ShortName "coll-exceed-limit-2"
+                 :EntryTitle "entry-title-exceed-limit-2"})
+               {:token "mock-echo-system-token"})
+        coll3 (data-core/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection
+                {:ShortName "coll-exceed-limit-3"
+                 :EntryTitle "entry-title-exceed-limit-3"})
+               {:token "mock-echo-system-token"})]
+     (testing "ingest on PROV1 with subscription request for subscriber-id below subscription limit"
+       (let [concept (subscription-util/make-subscription-concept {:provider-id "PROV1"
+                                                                   :Name "Sub1"
+                                                                   :CollectionConceptId (:concept-id coll1)
+                                                                   :SubscriberId "user1"
+                                                                   :EmailAddress "foo@example.com"})
```

Comment:
We can drop `:provider-id`, `:EmailAddress`, and `:Name` from each call to `make-subscription-concept` as they aren't necessary/relevant for the test.

Reply:
I dropped most of the fields, but I found when I dropped :Name as well it would appear no longer create new subscriptions but rather modify previous subscriptions. Without :Name the tests to create a subscription returned code 200 OK, whereas they properly returned 201 Created with :Name specified.